docs/discovery/bos/intro.md

```diff
@@ -0,0 +1,31 @@
+---
+sidebar_position: 1
+---
+
+# Overview
```

Comment:
Maybe this can became the BOS-docs landing page. The only problem is that it has too much text.

Reply:
We could leave the landing page, and add this as an "Overview" article under `Home`